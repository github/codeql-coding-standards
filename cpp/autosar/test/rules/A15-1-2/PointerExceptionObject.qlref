rules/A15-1-2/PointerExceptionObject.ql