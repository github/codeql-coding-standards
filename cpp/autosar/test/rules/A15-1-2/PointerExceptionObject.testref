cpp/common/test/rules/exceptionobjecthavepointertype/ExceptionObjectHavePointerType.ql