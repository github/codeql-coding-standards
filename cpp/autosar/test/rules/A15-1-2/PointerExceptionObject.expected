| test.cpp:9:11:9:12 | & ... | Exception object with pointer type ClassA * is thrown here. |
| test.cpp:11:11:11:12 | a2 | Exception object with pointer type ClassA * is thrown here. |
